src/prob/mcdcopf.jl

```diff
@@ -3,7 +3,7 @@ export solve_mcdcopf
 ""
 function solve_mcdcopf(file::String, model_type::Type, solver; kwargs...)
     data = _PM.parse_file(file)
-    process_additional_data!(data)
+    PowerModelsMCDC.build_mc_data!(data)
```

Comment:
```suggestion
    build_mc_data!(data)
```
We are into PowerModelsMCDC here: no need to specify the package.